android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java

```diff
@@ -62,15 +58,56 @@ public String getName() {
 
     @Override
     protected ImageViewWithUrl createViewInstance(ThemedReactContext reactContext) {
-        requestManager = Glide.with(reactContext);
         return new ImageViewWithUrl(reactContext);
     }
 
+    private static RequestListener<GlideUrl, GlideDrawable> LISTENER = new RequestListener<GlideUrl, GlideDrawable>() {
+        @Override
+        public boolean onException(
+                Exception e,
+                GlideUrl uri,
+                Target<GlideDrawable> target,
+                boolean isFirstResource
+        ) {
+            OkHttpProgressGlideModule.forget(uri.toStringUrl());
+            if (!(target instanceof ImageViewTarget)) {
+                return false;
+            }
+            ImageViewWithUrl view = (ImageViewWithUrl) ((ImageViewTarget) target).getView();
+            ThemedReactContext context = (ThemedReactContext) view.getContext();
+            RCTEventEmitter eventEmitter = context.getJSModule(RCTEventEmitter.class);
```

Comment:
Why use `RCTEventEmitter` and not `RCTNativeAppEventEmitter` ? 
Maybe it is right but I'm not sure why. Most modules tends to use either `RCTDeviceEventEmitter` or recently `RCTNativeAppEventEmitter`